CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 0.3.0-dev.0
```

Comment:
Shouldn't this be  0.3.0-nullsafety.0?

Reply:
Does the nullsafety tag imply it works with the latest beta? Or is it fine that it requires a newer dev-release?

Reply:
Its just a convention. Any version with a suffix in it, is a prerelease: https://dart.dev/tools/pub/publishing#publishing-prereleases